sys-libs/readline/readline-6.3.8.recipe

```diff
@@ -85,6 +89,7 @@ BUILD()
 INSTALL()
 {
 	make install
+	strip -s $libDir/libhistory.so.6.3 $libDir/libreadline.so.6.3
```

Comment:
I meant replacing "strip" in the highlighted green line (line 92) with "strip$secondaryArchSuffix". this is to use the target architecture version of strip.